README.md

```diff
@@ -25,38 +61,172 @@ By using this handler, you can now use CDK templates with all your favorite Scep
 
 The template "type" for this handler is `cdk`. 
 
+### Deployment Types
+The CDK Handler supports two different deployment types, which function somewhat differently. These
+are "bootstrapped" and "bootstrapless", both passed to the `deployment_type` template handler argument.
+
+#### The "bootstrapped" deployment_type
+The "bootstrapped" deployment type is a more typical CDK-like way to deploy infrastructure and 
+conforms more to the standard ways _CDK_ operates. If your organization has a lot of CDK infrastructure,
+using this deployment_type likely will allow Sceptre to interoperate with existing patterns and 
+policies. 
+
+The `"bootstrapped"` deployment_type causes Sceptre assets (namely S3-destined files and 
+ECR-destined images) to be deployed using the usual CDK-bootstrapped machinery. Specifically, by 
+referencing a "qualifier", CDK looks for a corresponding stack in the AWS account that contains the
+specific S3 bucket and ECR repo, as well as IAM roles to be assumed in order build and push those
+assets up to the cloud.
+
+In order to use the "bootstrapped" deployment type to push assets to the cloud, a CDK bootstrap stack 
+with matching qualifier must already be deployed. It may be deployed via CDK (outside of Sceptre) or
+you can use CDK to generate the bootstrap template for Sceptre to deploy using  
```

Comment:
I'm confused by the statement `In order to use the "bootstrapped" deployment type to push assets to the cloud, a CDK bootstrap stack with matching qualifier must already be deployed.`.  Doesn't the bootrapped mode do the  "bootstrap stack with matching qualifier"?

Reply:
No it doesn't. You need to either _generate_ the bootstrap stack using `cdk bootstrap --show-template` and turn that into a Sceptre-deployed stack template. Or you need to deploy the CDK bootstrap stack via CDK. The `bootstrapped` deployment_type will use that stack, however it's deployed, so long as you specify a qualifier that matches the one the bootstrap stack was deployed with.

Reply:
ok, i think i'm just confused because i just don't have enough experience with using the CDK.